src/Identity/Core/src/Microsoft.AspNetCore.Identity.csproj

```diff
@@ -9,6 +9,7 @@
     <IsPackable>false</IsPackable>
     <IsTrimmable>true</IsTrimmable>
     <EnableRequestDelegateGenerator>true</EnableRequestDelegateGenerator>
+    <Features>$(Features);InterceptorsPreview</Features>
```

Comment:
I thought the SDK did this for us automatically?

Reply:
The SDK only does this for projects using the WebSDK (which is why we don't need it in the minimal sample) at the moment, so it still needs to be configured in library projects like this.

We can probably move the insertion lower in the SDK, and may need to for the config binder.

cc: @layomia